includes/wizards/class-subscriptions-wizard.php

```diff
@@ -37,6 +39,327 @@ class Subscriptions_Wizard extends Wizard {
 	 */
 	protected $capability = 'edit_products';
 
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		parent::__construct();
+
+		add_action( 'rest_api_init', [ $this, 'register_api_endpoints' ] );
+		add_filter( 'woocommerce_product_data_store_cpt_get_products_query', [ $this, 'handle_only_get_newspack_subscriptions_query' ], 10, 2 );
```

Comment:
The Subscriptions Wizard only displays and manages products created through the Subscriptions Wizard. I think that's the way to go because it would be super confusing/breaking if we pulled in swag or other such products. This should keep things working nicely.